### PR TITLE
MAINT: Conversion of some strings to fstrings, part III

### PR DESCRIPTION
**Pull request by jakobjakobson13**

### CODE REVIEW

File: numpy/ma/mrecords.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def _checknames(descr, names=None):
         elif isinstance(names, str):
             new_names = names.split(',')
         else:
-            raise NameError("illegal input names %s" % repr(names))
+            raise NameError(f'illegal input names {repr(names)}')
```
Comment (BvB93) [inline numpy/ma/mrecords.py:63]:
```suggestion
            raise NameError(f'illegal input names {names!r}')
```

File: numpy/ma/tests/test_old_ma.py [hunk 2]
```diff
@@ -27,7 +27,7 @@
 def eq(v, w, msg=''):
     result = allclose(v, w)
     if not result:
-        print("Not eq:%s\n%s\n----%s" % (msg, str(v), str(w)))
+        print(f'Not eq:{msg}\n{str(v)}\n----{str(w)}')
```
Comment (BvB93) [inline numpy/ma/tests/test_old_ma.py:30]:
```suggestion
        print(f'Not eq:{msg}\n{v}\n----{w}')
```

File: numpy/ma/timer_comparison.py [hunk 3]
```diff
@@ -94,7 +93,7 @@ def assert_array_compare(self, comparison, x, y, err_msg='', header='',
                 reduced = reduced.tolist()
             if not cond:
                 match = 100-100.0*reduced.count(1)/len(reduced)
-                msg = build_err_msg([x, y],
+                msg = build_err_msg([x, y],
```
Comment (BvB93) [inline numpy/ma/timer_comparison.py:96]:
```suggestion
                msg = build_err_msg([x, y],
```
Trailing whitespace.

**Comment by mattip** [comment]
Thanks for the desire and effort to make NumPy better.

At some point we had discussed reformatting PRs and decided against accepting these. If this is part of some other code change, converting regular formatting to f-strings is fine, but otherwise we would prefer not to spend reviewer time and CI resources on PRs like this.

I guess we should put it in the dev guide somewhere, or otherwise make it clear, sorry.

**Pull request merged**